src/pythonturtle.py

```diff
@@ -2,23 +2,25 @@
 Main module which defines ApplicationWindow,
 the main window of PythonTurtle.
 """
+import multiprocessing
+
 import wx
 import wx.aui
 import wx.lib.buttons
 import wx.lib.scrolledpanel
 
-from customscrolledpanel import CustomScrolledPanel
+import homedirectory
 import shelltoprocess
-import turtlewidget
 import turtleprocess
-import multiprocessing
-import homedirectory; homedirectory.do()
```

Comment:
@cool-RR This is really bad style. There must be a better way to compute and deal with the "current working directory" and such. (See [related flake8 complaint](https://travis-ci.org/cool-RR/PythonTurtle/jobs/429498989#L584).)

Currently, this is unsolved. I think we have to refactor the project, though, to improve the situation.

Also, the project - [from Python's point of view](https://travis-ci.org/cool-RR/PythonTurtle/jobs/429499000#L706) - is currently called `src`. This is due to `__init__.py` being in the `src` folder. No good. This asks for being fixed.

Reply:
Maybe we can do some collaborative programming to solve that in #106. Would be nice!